README.md

```diff
@@ -30,53 +42,72 @@ Your app code should look something like this:
+        })();
     </script>
 
-You can find your Project-ID in the Phrase Translation Center.
+You can find your Project-ID and Account-ID in the Phrase Translation Center.
 
+Old version of the ICE is not available since version 2.0.0. If you still would rather use the old version, please go back to 1.x.x versions.
 
-## Using the US Datacenter with ICE
+#### Using the US Datacenter with ICE
 
-In addition to `projectId` in the config, also add the US specific URLs to enable working through the US endpoint.
-```
-    baseUrl: "https://us.app.phrase.com",
-    apiBaseUrl: 'https://api.us.app.phrase.com/api/v2',
-    oauthEndpointUrl: "https://api.us.app.phrase.com/api/v2/authorizations",
-    profileUrl: "https://us.app.phrase.com/settings/profile",
-```
+In addition to `projectId` and `accountId` in the config, also add the US specific datacenter setting to enable working through the US endpoint.
 
-e.g
 ```
 window.PHRASEAPP_CONFIG = {
     projectId: "YOUR-PROJECT-ID",
-    baseUrl: "https://us.app.phrase.com",
-    apiBaseUrl: 'https://api.us.app.phrase.com/api/v2',
-    oauthEndpointUrl: "https://api.us.app.phrase.com/api/v2/authorizations",
-    profileUrl: "https://us.app.phrase.com/settings/profile",
+    accountId: "YOUR-ACCOUNT-ID",
+    datacenter: "us",
 };
 ```
 
-## Usage
+### How does it work
+
+Set the `PHRASEAPP_ENABLED` to `True/False` to enable or disable In-Context-Editing. When set to False, it will fall back to standard Flask-Babel's gettext functions. Disable Phrase for production environments at any time!
+
+When `PHRASEAPP_ENABLED = True` this package modifies the returning values from translation functions to present a format which the ICE can read.
+
+Flask-Phrase provides In-Context translating facilities to your Flask app by hooking into [flask-babel's](https://pypi.org/project/flask-babel/) gettext function. It exposes the underlying key names to the JavaScript editor that is provided by Phrase.
+
+### Test
+
+Run unit tests:
+
+```bash
+python manage.py test
+```
+
+## :white_check_mark: Commits & Pull Requests
+
+We welcome anyone who wants to contribute to our codebase, so if you notice something, feel free to open a Pull Request! However, we ask that you please use the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification for your commit messages and titles when opening a Pull Request.
+
+Example: `chore: Update README`
+
+## :question: Issues, Questions, Support
+
+Please use [GitHub issues](https://github.com/phrase/Flask-Phrase/issues) to share your problem, and we will do our best to answer any questions or to support you in finding a solution.
 
-Set the PHRASEAPP_ENABLED to True/False to enable or disable In-Context-Editing. When set to False, it will fall back to standard Flask-Babel's gettext functions. Disable Phrase for production environments at any time!
+## resources
```

Comment:
This could be capitalized
```suggestion
## Resources
```

and maybe an emoji in the front?  😁 
```suggestion
## :books: Resources
```

Reply:
:D